dynawo/sources/API/CRT/xsd/criteria.xsd

```diff
@@ -30,13 +30,18 @@
     <xs:sequence>
       <xs:element name="parameters" type="dyn:CriteriaParams" minOccurs="1" maxOccurs="1"/>
       <xs:element name="component" minOccurs="0" maxOccurs="unbounded" type="dyn:Component"/>
+      <xs:element name="country" minOccurs="0" maxOccurs="unbounded" type="dyn:Country"/>
     </xs:sequence>
   </xs:complexType>
 
   <xs:complexType name="Component">
     <xs:attribute name="id" use="required" type="xs:string"/>
   </xs:complexType>
 
+  <xs:complexType name="Country">
+    <xs:attribute name="id" use="required" type="xs:string"/>
+  </xs:complexType>
+
```

Comment:
Perhaps to be done later on, but do we want to close the possibilities here? Indeed, do we want to have a restriction of the string type to a few enum values?

Reply:
We could add later the possibility to have regular expression too, but I guess the main usage of this is to limit to the main country simulated so I guess this MR covers most of the needs right?

Reply:
Indeed, it covers most of the needs for sure.
In addition, I just checked if the country attribute is limited in IIDM to some values but apparently not so we can definetely let it this way.